website_sale_checkout_skip_payment/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+==================================
+Website Sale Checkout Skip Payment
+==================================
+
+This module allows to logged users to checkout with no payment step. At the
+end of the checkout proccess the quotation is sent to the user email address
+and set to *Qoutation Sent* state.
+
+Usage
+=====
+
+You can deactivate this option for a particular partner unchecking
+"Skip Website Checkout Payment" field.
+
```

Comment:
Remember that if you customer is not registered it will go though the standard process